.github/workflows/IssuesCloseStale.yml

```diff
@@ -0,0 +1,16 @@
+name: Close Stale Issues
+on:
+  schedule:
+    - cron: '00 0 * * *'
+
+jobs:
+  close_stale_issues:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Close stale issues
+        uses: actions/stale@v8
+        with:
+          stale-issue-message: 'This issue is stale because it has been open 90 days with no activity. Remove stale label or comment or this will be closed in 30 days.'
```

Comment:
we should think about the exact message here still, like link to the support page once we have it.